packages/fx-core/src/component/utils/common.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { DeployPrerequisiteError } from "../error/deployError";
+import { exec } from "child_process";
+
+/**
+ * check parameter, throw error if value is null or undefined
+ * @param name parameter name
+ * @param value parameter value
+ */
+export function checkMissingDeployArgs<T>(name: string, value: T | null | undefined): T {
+  if (!value) {
+    throw DeployPrerequisiteError.somethingMissing(name);
```

Comment:
Since it throws deploy specific error, is it better to be moved to component/deploy folder?

Reply:
Yes, maybe we should throw a more common error here.